src/composables/channel.ts

```diff
@@ -147,6 +148,35 @@ export const useChannel = defineStore('channel', () => {
     socket?.emit('update-handSignal', { handSignal: student.handSignal });
   }
 
+  function updatePermission(studentId: string): void {
+    console.log(studentId)
+    const student = studentById(studentId);
+    console.log(student)
```

Comment:
`console.log`s gehören nicht in source code. Die kann man zum debuggen verwenden, beim pushen aber aufpassen, dass man diese wieder entfernt